.changeset/cool-days-look.md

```diff
@@ -0,0 +1,7 @@
+---
+'@astrojs/starlight': patch
```

Comment:
I think this counts as a minor! It’s essentially adding a feature, and if we’re changing some of our set-up code, I guess theoretically could even break the odd person in a rare case.

```suggestion
'@astrojs/starlight': minor
```